dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -25,8 +25,11 @@ disableJspRuntimeCompilation=Disable compilation of JSPs at runtime.
 extendedDocumentRoot.name=Extended document root
 extendedDocumentRoot=Directory that the JSP engine will search for additional JSP files to serve.
 
+javaSourceLevel.name=Java source level
+javaSourceLevel=Default Java source level for JSPs compiled by the JSP engine.
+
 jdkSourceLevel.name=JDK source level
-jdkSourceLevel=Default Java source level for JSPs compiled by the JSP engine.
+jdkSourceLevel=Deprecated. Use javaSourceLevel instead. Default Java source level for JSPs compiled by the JSP engine.
```

Comment:
Since I changed this, 
1) I'll need new approval / review
2 I should remove the translations since they'll need to be redone.

Reply:
Use stabilized instead? See what we should say here.  Add what levels are supported?